frontend/src/app/projects/dashboard/metrics/page.tsx

```diff
@@ -0,0 +1,296 @@
+  const orderingParam = searchParams.get('order')
+  const currentFilterKeys = []
+  if (healthFilter) {
+    currentFilters = {
+      ...healthFiltersMapping[healthFilter],
+    }
+    currentFilterKeys.push(healthFilter)
+  }
+  if (levelFilter) {
+    currentFilters = {
+      ...currentFilters,
+      ...levelFiltersMapping[levelFilter],
+    }
+    currentFilterKeys.push(levelFilter)
+  }
+  if (orderingParam) {
+    currentOrdering = orderingMapping[orderingParam] || currentOrdering
+  }
+
+  const [metrics, setMetrics] = useState<HealthMetricsProps[]>([])
+  const [metricsLength, setMetricsLength] = useState<number>(0)
+  const [pagination, setPagination] = useState({ offset: 0, limit: PAGINATION_LIMIT })
+  const [filters, setFilters] = useState(currentFilters)
+  const [ordering, setOrdering] = useState(currentOrdering)
+  const [activeFilters, setActiveFilters] = useState(currentFilterKeys)
+  const [activeOrdering, setActiveOrdering] = useState(
+    Object.keys(orderingMapping).filter(
+      (key) => JSON.stringify(orderingMapping[key]) === JSON.stringify(currentOrdering)
+    ) || ['scoreDESC']
+  )
+  const {
+    data,
+    error: graphQLRequestError,
+    loading,
+    fetchMore,
+  } = useQuery(GET_PROJECT_HEALTH_METRICS_LIST, {
+    variables: {
+      filters,
+      pagination: { offset: 0, limit: PAGINATION_LIMIT },
+      ordering: Object.values(ordering),
+    },
+  })
+
+  useEffect(() => {
+    if (data) {
+      setMetrics(data.projectHealthMetrics)
+      setMetricsLength(data.projectHealthMetricsDistinctLength)
+    }
+    if (graphQLRequestError) {
+      handleAppError(graphQLRequestError)
+    }
+  }, [data, graphQLRequestError])
+
+  const orderingSections: DropDownSectionProps[] = [
+    {
+      title: 'Score',
+      items: [
+        { label: 'High to Low', key: 'scoreDESC' },
+        { label: 'Low to High', key: 'scoreASC' },
+      ],
+    },
+  ]
+  const filteringSections: DropDownSectionProps[] = [
+    {
+      title: 'Project Level',
+      items: [
+        { label: 'Incubator', key: 'incubator' },
+        { label: 'Lab', key: 'lab' },
+        { label: 'Production', key: 'production' },
+        { label: 'Flagship', key: 'flagship' },
+      ],
+    },
+    {
+      title: 'Project Health',
+      items: [
+        { label: 'Healthy Projects', key: 'healthy' },
+        { label: 'Projects Needing Attention', key: 'needsAttention' },
+        { label: 'Unhealthy Projects', key: 'unhealthy' },
+      ],
+    },
+    {
+      title: 'Reset Filters',
+      items: [{ label: 'Reset All Filters', key: 'reset' }],
+    },
+  ]
+
+  const getCurrentPage = () => {
+    return Math.floor(pagination.offset / PAGINATION_LIMIT) + 1
+  }
+
+  return (
+    <>
+      <div className="mb-4 flex items-center justify-between">
+        <h1 className="text-2xl font-bold">Project Health Metrics</h1>
+        <div className="flex items-center space-x-2">
+          <ProjectsDashboardDropDown
```

Comment:
Also, just an idea and I know we've been through this already, but would like to reiterate still.
Whaat do you think of this - what if we split these into 2? I can see how this could be cleaner look if we have 2 separate dropdowns for Level and Health. I hardcoded some things to show how that might look like:
<img width="417" height="77" alt="Screenshot 2025-07-15 at 6 03 56 PM" src="https://github.com/user-attachments/assets/a26399a7-443b-418b-bdf6-7bcf6e12f490" />
Would that be too much trouble to implement it this way? 🤔